src/internal/operators/share.ts

```diff
@@ -53,8 +64,8 @@ export function share<T>(options: ShareConfig<T>): MonoTypeOperatorFunction<T>;
 /**
  * Returns a new Observable that multicasts (shares) the original Observable. As long as there is at least one
  * Subscriber this Observable will be subscribed and emitting data. When all subscribers have unsubscribed it will
- * unsubscribe from the source Observable. Because the Observable is multicasting it makes the stream `hot`.
- * This is an alias for `multicast(() => new Subject()), refCount()`.
```

Comment:
Since `multicast` and `refCount` are deprecated, this can be removed as irrelevant.